Documentation/Getting-Started/Prerequisites/authenticated-registry.md

```diff
@@ -4,7 +4,7 @@ title: Authenticated Container Registries
 
 If you want to use an image from authenticated docker registry (e.g. for image cache/mirror), you'll need to
 add an `imagePullSecret` to all relevant service accounts. This way all pods created by the operator (for service account:
```

Comment:
Let's remove the sentence that mentions the specific service accounts here, and just refer them to the next section.
```suggestion
add an `imagePullSecret` to all relevant service accounts. See the next section for the required service accounts.
```